app/assets/stylesheets/helpers/_reset.scss

```diff
@@ -1,29 +1,115 @@
 // flatten browser default padding and font-sizing so we can cleanly apply our own
 
-/* Borrowed from http://meyerweb.com/eric/tools/css/reset/ */
-div, span,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed,
-figure, figcaption, footer, header, hgroup,
-menu, nav, output, ruby, section, summary,
-time, mark {
+// Borrowed from http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Ok. It looks like the `ul` margins aren't reset – feels like the problem that was had with the breadcrumbs component. Must be somewhere else in static that's doing that part of the reset.

Let's not worry about it for now. Thanks for looking into it.